src/Native/System.Net.Security.Native/pal_gssapi.cpp

```diff
@@ -185,6 +185,36 @@ extern "C" uint32_t NetSecurityNative_InitSecContext(uint32_t* minorStatus,
     return NetSecurityNative_HandleError(majorStatus, &gssBuffer, outBuffer);
 }
 
+extern "C" uint32_t NetSecurityNative_AcceptSecContext(uint32_t* minorStatus,
+                                                     GssCtxId** contextHandle,
+                                                     uint8_t* inputBytes,
+                                                     uint32_t inputLength,
+                                                     struct PAL_GssBuffer* outBuffer)
+{
+    assert(minorStatus != nullptr);
+    assert(contextHandle != nullptr);
+    assert(outBuffer != nullptr);
+    assert(inputBytes  != nullptr || inputLength == 0);
+
+
+    GssBuffer inputToken {.length = UnsignedCast(inputLength), .value = inputBytes};
+    GssBuffer gssBuffer { .length = 0, .value = nullptr };
```

Comment:
Nits: a few formatting things, e.g. extra blank line above, the two initializers not having the same spacing around the fields, etc.